client/config/cmd.go

```diff
@@ -29,12 +29,13 @@ func Cmd() *cobra.Command {
 func runConfigCmd(cmd *cobra.Command, args []string) error {
 
 	clientCtx := client.GetClientContextFromCmd(cmd)
-
 	configPath := filepath.Join(clientCtx.HomeDir, "config")
 
-	if err := ensureConfigPath(configPath); err != nil {
-		return fmt.Errorf("couldn't make client config: %v", err)
-	}
+	/*
```

Comment:
I think we dont need ensureConfigPath anymore, cause ~/.simapp/config directory will exist in any way due to ReadFromClientConfig logic